support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -3,174 +3,84 @@ package controllers
 import actions.CustomActionBuilders
 import admin.settings.{AllSettings, AllSettingsProvider, SettingsSurrogateKeySyntax}
 import assets.{AssetsResolver, RefPath, StyleContent}
-import cats.implicits._
-import com.gu.i18n.Country.UK
-import com.gu.i18n.CountryGroup
-import com.gu.identity.model.{User => IdUser}
-import com.gu.monitoring.SafeLogger
-import com.gu.monitoring.SafeLogger._
 import com.gu.support.catalog.DigitalPack
-import com.gu.support.config.{PayPalConfigProvider, Stage, StripeConfigProvider}
 import com.gu.support.encoding.CustomCodecs._
 import com.gu.support.pricing.{PriceSummaryServiceProvider, ProductPrices}
-import com.gu.support.promotions.{DefaultPromotions, ProductPromotionCopy, PromoCode, PromotionCopy, PromotionServiceProvider}
-import config.{RecaptchaConfigProvider, StringsConfig}
-import controllers.UserDigitalSubscription.{redirectToExistingThankYouPage, userHasDigitalSubscription}
-import play.api.libs.circe.Circe
+import com.gu.support.promotions._
+import com.gu.support.zuora.api.ReaderType.{Direct, Gift}
+import config.StringsConfig
 import play.api.mvc._
 import play.twirl.api.Html
-import services._
 import views.EmptyDiv
 import views.ViewHelpers._
-import views.html.helper.CSRF
-import views.html.subscriptionCheckout
-import com.gu.support.zuora.api.ReaderType.{Direct, Gift}
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 class DigitalSubscriptionController(
   priceSummaryServiceProvider: PriceSummaryServiceProvider,
   landingCopyProvider: LandingCopyProvider,
   val assets: AssetsResolver,
   val actionRefiners: CustomActionBuilders,
-  identityService: IdentityService,
-  testUsers: TestUserService,
-  membersDataService: MembersDataService,
-  stripeConfigProvider: StripeConfigProvider,
-  payPalConfigProvider: PayPalConfigProvider,
   components: ControllerComponents,
   stringsConfig: StringsConfig,
```

Comment:
I'm not quite sure why we have StringsConfig, it just seems to indirectly read from the file referenced below in github?  Unless this was the precursor to the equivalent of the landingCopyProvider.
https://github.com/guardian/support-frontend/blob/6cebfa09877911211ad7e266d8dded59bd018562/support-frontend/conf/strings.conf#L1-L6

Reply:
Hmm, no idea. Perhaps we can remove it

Reply:
let's do that "one day"